docs/versioned-plugins/inputs/beats-v4.0.1.asciidoc

```diff
@@ -46,57 +46,57 @@ output {
 
 NOTE: The Beats shipper automatically sets the `type` field on the event.
 You cannot override this setting in the Logstash config. If you specify
-a setting for the <<plugins-inputs-beats-type,`type`>> config option in
+a setting for the <<{version}-plugins-inputs-beats-type,`type`>> config option in
```

Comment:
This is an internal link, so it needs to include {version}. We'll need to fix the script to add {version} to all internal links. However, it might make more sense to do this *after* we've converted the format of all external links in asciidoc source in the various repos. It's going to be a bit of cleanup work, and probably manual.